docs/src/writing-tests-js-python.md

```diff
@@ -177,6 +242,29 @@ test.describe('navigation', () => {
 });
 ```
 
+### Using Test Hooks
+* langs: python
+
+You can use various [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request) to execute code before or after your tests and to share objects between them. A `function` scoped fixture e.g. with autouse behaves like a beforeEach/afterEach. And a `module` scoped fixture with autouse behaves like a beforeAll/afterAll which runs before all and after all the tests.
+
+```python
+import pytest
+from playwright.sync_api import Page
+
+
+@pytest.fixture(scope="function", autouse=True)
+def before_each_after_each(page: Page):
+    print("beforeEach")
```

Comment:
```suggestion
    print("before the test runs, aka. before each")
```

Not strong on this one, do you think it helps folks to have it a bit more verbose inside the print statement?

Reply:
yeh i like it. they can always change it as its just an example so why not especially at getting started stage